src/EdFi.AnalyticsMiddleTier.DataStandard31/Asmt/MSSQL/0014-View-StudentAssessmentFact-Create.sql

```diff
@@ -0,0 +1,126 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+IF EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.VIEWS WHERE TABLE_SCHEMA = 'analytics' AND TABLE_NAME = 'asmt_StudentAssessmentFact')
+BEGIN
+	DROP VIEW analytics.asmt_StudentAssessmentFact
+END
+GO
+
+CREATE VIEW analytics.asmt_StudentAssessmentFact AS
+
+	SELECT
+        CONCAT(StudentAssessment.AssessmentIdentifier, '-', StudentAssessment.Namespace, '-', StudentAssessment.StudentAssessmentIdentifier, '-', StudentAssessment.StudentUSI) AS StudentAssessmentKey,
+		CASE WHEN SASOA.StudentUSI IS NULL 
```

Comment:
Not related with this ticket. but this case statement makes me think that there could be a bug in **analytics.asmt_AssessmentFact**. Because there are concatenations of values that could be null. And there is no validation to check that. We could need a bug ticket to fix it.